packages/monorepo-tools/rewrite_history_into.sh

```diff
@@ -12,6 +12,12 @@
 SUBDIRECTORY=$1
 REV_LIST_PARAMS=${@:2}
 echo "Rewriting history into a subdirectory '$SUBDIRECTORY'"
+
```

Comment:
Was this change necessary? rewrite_history_into.sh is used only during the build of monorepo. Why you backup and restore tags during the build?

Reply:
We agreed on this during PIA.
Since git do not backup tags during `git filter-branch`, we decided to do so even during creating monorepo.
Behavior before this change is strange - git does `filter-branch`, moves tags. Then git is reseted to original state, but tags are still on commits after rewriting history into. Then second package is processed, but tags still aims to first package commits. And in the end `filter-branch` deals with this strange situation "somehow".
This change makes this strange situation more straightforward - backup all tags, filter-branch, restore all tags.

Reply:
this comment is invalid. Change reverted from rewrite_history_into.sh